Search/SearchPlugins/zh-HK/wikipedia-zh-TW.xml

```diff
@@ -1,25 +0,0 @@
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Are you able to explain why we're removing a few of these XML files? Maybe I'm missing something :)

Reply:
It seems that running the scrape_plugins script deletes these files (it removes and recreates the entire SearchPlugins directory at the start). To verify, I checked out master and ran the script and it resulted in the same modified/deleted xml files.